### DISCUSSION TITLE
Preventing multiple bookmarks

### DISCUSSION THREAD

**Discussion by WebworkrNet** [floccusaddon/floccus#1271] [Q&A] [answered]
Currently, the Floccus app (for Android) does not prevent existing bookmarks from being created again, i.e. twice or more. Is this deliberate?

I have not found anything on this topic under Discussion, FAQs (floccus.org), and Issues.

Thanks

**Comment by marcelklehr** [comment]
Hey @WebworkrNet

Thanks for your feedback!

> Is this deliberate?

Heh, not really. Adding a note to the bookmark creation screen would indeed, be nice :) Can you open a new issue?

**Comment by WebworkrNet** [comment, marked as answer]
#1278
